test/integration.html

```diff
@@ -12,8 +12,20 @@
 <head>
   <title>iron-location</title>
 
-  <script src="../../webcomponentsjs/webcomponents.js"></script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
   <script src="../../web-component-tester/browser.js"></script>
+  <script>
```

Comment:
this would be a breaking change to the url polyfill. As far as I know this is not documented anywhere as a standard